frontend/src/router/index.js

```diff
@@ -57,6 +58,9 @@ const routes = [
         path: 'checkEligibility',
         name: 'CheckEligibility',
         component: CheckEligibility,
+        beforeEnter: (to, _, next) => {
```

Comment:
does _ have specific meaning with vue router? didn't see it in any docs.

Reply:
It's used a hint to linters/formatters that the parameter is unused but the developer is aware of it. In this case you can't access next (required) without also accepting the second parameter (which is unused).